src/lizaalert/users/models.py

```diff
@@ -321,3 +331,25 @@ def create_default_volunteer_level(sender, instance, created, **kwargs):
 
             if not existing_record:
                 VolunteerLevel.objects.create(volunteer=volunteer, level=beginner_level, confirmed=True)
+
+
+class UserDivisionLevel(models.Model):
+    """Модель для связи пользователей, направлений и уровней."""
+
+    volunteer = models.ForeignKey(
+        Volunteer,
+        on_delete=models.CASCADE,
+        related_name="divisionlevel",
```

Comment:
```suggestion
        related_name="division_level",
```

```

# Bad Example
myvariable = 42  # Not following naming style
```

```

# Good Example
my_variable = 42  # Consistent and descriptive name
```

https://medium.com/@rowainaabdelnasser/python-naming-conventions-10-essential-guidelines-for-clean-and-readable-code-fe80d2057bc9#:~:text=For%20class%20names%20in%20Python,clarity%20and%20improve%20code%20comprehension.